components/manager/scripts/validate.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
We have tons of other operations done in the blueprint rather than in the CLI. We should have some sort of a criteria what type of functionality exists in the UI and what's in the blueprint. For validation, I'd prefer the blueprint because I don't want to depend on the CLI to perform validation — maybe in the future we'd like to allow bootstrapping though means other than the CLI.